spec.md

```diff
@@ -202,6 +232,48 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
           "5 TB storage",
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "properties": {
+                "billing-account": {
+                  "description": "Billing account number used to charge use of shared fake server.",
+                  "type": "string"
+                }
+              }
+            }
+          },
+          "update": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "properties": {
+                "billing-account": {
+                  "description": "Billing account number used to charge use of shared fake server.",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "create": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "properties": {
+                "billing-account": {
```

Comment:
This repetition of properties is something that will strike people as odd and I'd prefer if we dealt with it head-on. In particular, I think we should add a sentence someplace that makes it clear that brokers need to duplicate the properties on ALL methods and not try to take short-cuts like only putting them on "create" and assume people will carry that info over to the other methods.  Without this some people might take undocumented short-cuts and that will lead to interop issues.

Alternatively, we could provide a mechanism to optimize this and avoid the duplication but I suspect it might get messy to do so.